components/sections/app-search.js

```diff
@@ -206,8 +206,8 @@ export default function AppSearch({ data }) {
           let step9 = "Use HEAL Common Data Elements to Collect Your Data"
           bucket[7] = { status: status9, step: step9, notes: notes9 }
           break
-        case "data_repositories_hdp":
-          let status11 = keyValue ? "green" : "red"
+        case "repository_metadata":
+          let status11 = keyValue.split(",").length > 0 ? "green" : "red"
```

Comment:
This is a really weird way of doing this but I feel like it works. The problem is the key (`repository_metadata`) isn't an actual JSON-valid array:

```
"repository_metadata": "[{'repository_name': 'Microphysiology Systems Database', 'repository_study_ID': ''}]"
```

I'm assuming it's some default (python dict?) serialization, but it works for this use case since we really only need to check if the array is empty (`"[]"`) or not. If we every want to extract the data, the endpoint should really be fixed.

It's worth noting that this seems to be an issue for the other fields in this endpoint. See `investigators_name` parsing later in this file.

Reply:
thanks for discovering this. i'd hate to get bruned by this later, after we've all forgotten about this weirdness, so i'll pass the word along and report back.